docs/references/subcommands/test.md

```diff
@@ -25,3 +25,33 @@ By default, `fossa test` displays issues in a human-readable format. To instead
 ```sh
 fossa test --json
 ```
+
+### Test against previous revision
+
+With `--diff <ARG>`, you can check for _only_ new issue comapred to some `<ARG>` revision. 
+
+```sh
+fossa test --diff revisionToCompare
+```
+
+#### Example
+
+```sh
+fossa test --revision 34021e --diff v2.0.0
+```
+
+This will only report issues that are present in `34021e` revision,
+BUT are not present in revision `v2.0.0`.
+
+For instance, 
+
+* If the revision `v2.0.0` has issue: `A`, and the revision `34021e` has issue `A`, 
+  * `fossa-cli` will report no new issues discovered, and will exit with status code of 0.
+  
+
+* If the revision `v2.0.0` has issue: `A`, and the revision `34021e` has 0 issues, 
+  * `fossa-cli` will report no new issues discovered, and will exit with status code of 0.
+
+
+* If the revision `v2.0.0` has issue: `A`, and the revision `34021e` has issues `A`, `B`, 
+  * `fossa-cli` will report issue `B`, and will exit with status code of 1.
```

Comment:
[optional] Reworded this to flow a little more nicely. In particular:
- Refactors to keep the focus on what FOSSA does, not on the user (e.g., removes "you").
- Refactors "will do" style statements in favor of "does" style statements, which are more clear and direct.

(Escaping the triple-backticks so that they don't break the suggestion block)

```suggestion
### Test for new issues compared to another revision

`--diff <REVISION>` configures FOSSA to only report new issues observed with the current revision that weren't already reported on the specified `<REVISION>`.

\```sh
fossa test --diff revisionToCompare
\```

#### Example

\```sh
fossa test --revision 34021e --diff v2.0.0
\```

This only reports issues that are present in `34021e` revision,
BUT are not present in revision `v2.0.0`.

For instance, 

* If the revision `v2.0.0` has issue: `A`, and the revision `34021e` has issue `A`, 
  * `fossa-cli` reports no new issues discovered and exits with status code of 0.
  

* If the revision `v2.0.0` has issue: `A`, and the revision `34021e` has 0 issues, 
  * `fossa-cli` reports no new issues discovered and exits with status code of 0.


* If the revision `v2.0.0` has issue: `A`, and the revision `34021e` has issues `A`, `B`, 
  * `fossa-cli` reports issue `B` and exits with status code of 1.
```